test/powershell/Language/Scripting/Trap.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+
+Describe "Test trap" -Tags "CI" {
+    Context "Line after exception should not be invoked" {
+        It "line after exception should not be invoked" {
+            $a = . {trap {"trapped"; continue;}; . {"hello"; throw "exception"; "world"}}
+            $a.Length | Should Be 2
```

Comment:
My bad. I converted the Lite tests to Pester tests, but didn't look closely to their assert messages. Fixed.